core/modules/util/Command.h

```diff
@@ -0,0 +1,91 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ *
+ *  @author: John Gates,
+ */
+
+#ifndef LSST_QSERV_UTIL_COMMAND_H_
+#define LSST_QSERV_UTIL_COMMAND_H_
+
+// System headers
+#include <condition_variable>
+#include <memory>
+#include <mutex>
+
+
+namespace lsst {
+namespace qserv {
+namespace util {
+
+/// Tracker provides an interface for indicating an action is complete.
+///
+class Tracker {
```

Comment:
Where is the implementation of this class?  I would expect to find it in Command.cpp, but there doesn't seem to be that file?